oci/layout/oci_dest_test.go

```diff
@@ -68,9 +68,26 @@ func TestPutManifestUnrecognizedMediaType(t *testing.T) {
 
 	err = ociDest.PutManifest([]byte(m))
 	require.Error(t, err)
-	assert.Contains(t, `Unrecognized manifest media type: "application/vnd.docker.distribution.manifest.v1+prettyjws"`, err.Error())
+	assert.Equal(t, `can't create an OCI manifest from Docker V2 schema 1 manifest`, err.Error())
```

Comment:
Yes; the users don’t even want to know about schema versions, let alone the pointless signatures.